install.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Open a connection via PDO to create a
+ * new database and table with structure.
+ *
+ */
+require "config.php";
+try
+{
+	$connection = new PDO("mysql:host=$host", $username, $password, $options);
+	$sql = file_get_contents("data/init.sql");
+	$connection->exec($sql);
+
+	echo "Database and table users created successfully.";
+}
+catch(PDOException $error)
+{
+	echo $sql . "<br>" . $error->getMessage();
+}
```

Comment:
This is the "Driver" of the database connection. It creates the connection to the database using the variables defined in config.php. Then the sql statements in init.sql are run, and the database with corresponding tables is created on phpmyadmin. I think I may be on the right track here and hopefully I just have the connection variables wrong and it'll be a quick fix.